code/web/basis/src/main/java/com/castlemock/web/basis/web/view/controller/AbstractViewController.java

```diff
@@ -106,6 +111,19 @@ public ModelAndView redirect(final String url){
         return new ModelAndView("redirect:/web" + url);
     }
 
+    /**
+     * Create a model and view that redirects the user to a specific url
+     * @param url The url that the user should be redirected to.
+     * @param ex the exception to log its root cause
+     * @return A model and view that redirects the user to a specific url
+     * @since 1.36
+     */
+    public ModelAndView redirect(final String url, Exception ex){
```

Comment:
Absolutely yes. 
I noticed that the exception is got swollen and the logs don't contain any helpful information about the root cause.

I can suspect that this might happen to any other exception while loading the WSDL file. (by didn't test this scenario)

Reply:
I just verified that any error is suppressed, I tried to enter an invalid URL and got nothing on the log

Reply:
Alright. Good find! Resolving this conversation 😄